docs/Common/Structs/index.md

```diff
@@ -427,9 +427,9 @@
 
 |Name|Type|Mandatory|Additional|Description|
 |:---|:---|:--------|:---------|:----------|
-|value|String|true|maxlength: 65535||
-|imageType|Common.ImageType|true|||
-|isTemplate|Boolean|false|||
+|value|String|true|maxlength: 65535|The path to the dynamic image stored on HU or the static binary image itself.<br>Note: There is no guarantee that the image reference is valid at the time it is received.|
+|imageType|Common.ImageType|true||Describes whether it is a static or dynamic image.|
```

Comment:
```suggestion
|imageType|Common.ImageType|true||Describes whether it is a `STATIC` or `DYNAMIC` image.|
```

Reply:
Probably will revert this change too, as it appears rather odd on the actual developer portal